include/aktualizr-lite/tuf/tuf.h

```diff
@@ -0,0 +1,105 @@
+#ifndef AKLITE_TUF_TUF_H_
+#define AKLITE_TUF_TUF_H_
+
+#include <iostream>
+#include <string>
+
+#include <boost/algorithm/string.hpp>
```

Comment:
I think many of these headers became not needed here after the last cleaning work.

Reply:
Fixed.